### DISCUSSION TITLE
I keep on getting ClientResponseError: 0

### DISCUSSION THREAD

**Discussion by Nv7-GitHub** [pocketbase/pocketbase#637] [Q&A]
Every time I make a request through the JS SDK, it prints
```
ClientResponseError 0: Something went wrong while processing your request.
```
In the console, although it returns the correct result for the request and nothing seems to be broken. Should I just ignore this?

In the network tab, for every request I send, there is one red request which doesn't seem to return anything (says "Provisional Headers shown"), followed by a correct request that has 200 OK, the method, etc.

**Comment by ganigeorgiev** [comment]
I'm not able to reproduce it.

You could try to inspect the `ClientResponseError` object. if `err.isAbort` is `true` then this probably is a result of auto cancellation (https://github.com/pocketbase/js-sdk#auto-cancellation), aka. maybe there is hidden async bug in your code that results in triggering the same request multiple times.

**Reply by Nv7-GitHub** [reply]
Theres nothing in the response, but I think I'll just ignore it since it doesn't seem to be affecting anything

**Reply by ganigeorgiev** [reply]
You could inspect the thrown error like this:

```js
try {
    await client.records.getList("test", 1, 50);
} catch (err) {
    console.log(err.isAbort);
}
```

_As mentioned, this may be a hidden async bug and my recommendation would be to add a log somewhere before the request call to ensure that there isn't something that gets triggered unnecessary._
